src/openforms/js/components/admin/form_design/registrations/objectsapi/AddressNlObjectsApiVariableConfigurationEditor.js

```diff
@@ -28,12 +28,12 @@ export const AddressNlEditor = ({
   variable,
   components,
   namePrefix,
-  isGeometry,
   index,
   mappedVariable,
   objecttype,
   objectsApiGroup,
   objecttypeVersion,
+  backendOptions,
```

Comment:
is this used? If it is, then older versions are broken, no?

Reply:
No, `backendOptions` was added by me only for using it in the editors that I need to access properties like the `transformToList` in the `SelectBoxesVariableConfigurationEditor`  (didn't want to pass along all the editors specific properties that may confuse us).

Reply:
yeah but you don't need to specify the props in the signature that aren't used. it's okay if more props are passed, but now we have an unused variable that will trip up linters/typescript in the future :)